python/lsst/ts/m2gui/controltab/tab_rigid_body_pos.py

```diff
@@ -33,7 +37,326 @@ class TabRigidBodyPos(TabDefault):
+
+        Returns
+        -------
+        `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout_position = QFormLayout()
+        for axis in self._get_axes():
+            if axis.startswith("r"):
+                layout_position.addRow(f"Rotation {axis} (urad):", self._position[axis])
+            else:
+                layout_position.addRow(f"Position {axis} (mm):", self._position[axis])
+
+        layout = QVBoxLayout()
+        layout.addLayout(layout_position)
+        layout.addWidget(self._buttons["save_position"])
+
+        return create_group_box("Current Position Relative to Home", layout)
+
+    def _create_group_home_position(self):
+        """Create the group of home position.
+
+        Returns
+        -------
+        `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QVBoxLayout()
+        layout.addWidget(self._buttons["go_to_home"])
+        layout.addWidget(self._buttons["set_home"])
+
+        return create_group_box("Home Position", layout)
+
+    def _create_group_move_relative(self):
+        """Create the group of relative movement to the current position.
+
+        Returns
+        -------
+        `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout_position = QFormLayout()
+        for axis in self._get_axes():
+            if axis.startswith("r"):
+                layout_position.addRow(
+                    f"Offset {axis} (urad):", self._target_position_relative[axis]
+                )
+            else:
+                layout_position.addRow(
+                    f"Offset {axis} (mm):", self._target_position_relative[axis]
+                )
+
+        layout = QVBoxLayout()
+        layout.addLayout(layout_position)
+        layout.addWidget(self._buttons["jog"])
+        layout.addWidget(self._buttons["clear_values_relative"])
+
+        return create_group_box("Move with Offset Relative to Current Position", layout)
+
+    def _create_group_move_absolute(self):
+        """Create the group of absolute movement to the home position.
+
+        Returns
+        -------
+        `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout_position = QFormLayout()
+        for axis in self._get_axes():
+            if axis.startswith("r"):
+                layout_position.addRow(
+                    f"Rotation {axis} (urad):", self._target_position_absolute[axis]
+                )
+            else:
+                layout_position.addRow(
+                    f"Position {axis} (mm):", self._target_position_absolute[axis]
+                )
+
+        layout = QVBoxLayout()
+        layout.addLayout(layout_position)
+        layout.addWidget(self._buttons["go_to_position"])
+        layout.addWidget(self._buttons["clear_values_absolute"])
+
+        return create_group_box("Move to Position Relative to Home", layout)
+
+    def _set_signal_position(self, signal_position):
```

Comment:
You don't need this to happen in a function. Connecting it in __init__ is fine, along some comment. This is a one time operation, it will not be called again..

Reply:
I prefer to keep this as a function to be consistent with other parts of code. In addition, if I add more field to that `signal_position` in a latter time, I will only need to update the implementation in `_set_signal_position()`. Thanks!